apps/pdc-frontend/src/styles/print.css

```diff
@@ -0,0 +1,165 @@
+@media print {
+  header img,
+  header svg,
+  .utrecht-logo,
+  [class*="logo"] {
+    display: block;
+    max-height: 50px; /* make Logo smaller? */
+    width: auto;
+  }
+
+  body {
+    word-wrap: break-word;
+    overflow-wrap: break-word;
+    hyphens: auto;
+  }
+
+  /* Headers size */
+  .utrecht-print-page {
+    --utrecht-heading-2-font-size: var(--utrecht-heading-4-font-size);
+    --utrecht-heading-3-font-size: var(--utrecht-heading-4-font-size);
+    --utrecht-heading-5-font-size: var(--utrecht-heading-4-font-size);
+    --utrecht-heading-6-font-size: var(--utrecht-heading-4-font-size);
+  }
+
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    display: block;
+    page-break-after: avoid;
+  }
+
+  /* Link visible */
+  a {
+    text-decoration: none;
+    color: var(--utrecht-color-block);
+  }
+
+  a[href]:after {
+    content: " (" attr(href) ")";
+    font-size: 0.8em;
+    font-weight: normal;
+    font-style: italic;
+  }
+
+  a[href^="#"]:after,
+  a[href^="javascript:"]:after {
+    content: "";
+  }
+
+  .utrecht-accordion__panel,
+  .utrecht-skip-link,
+  .utrecht-search-bar-wrapper,
+  .utrecht-kcm-survey {
+    display: none !important;
+  }
+
+  /* Accordion open visible */
+  .utrecht-accordion__panel {
+    display: block !important;
+    max-height: none !important;
+    overflow: visible !important;
+    visibility: visible !important;
+    opacity: 1 !important;
+  }
+
+  .utrecht-image {
+    display: block;
+    max-inline-size: 500px;
+    page-break-inside: avoid;
+  }
+
+  ol {
+    counter-reset: item;
+  }
+
+  ol > li {
+    counter-increment: item;
+  }
+
+  ol ol > li {
+    display: block;
+  }
+
+  ol ol > li::before {
+    content: counters(item, ".") ". ";
+    margin-block-start: -20px;
+  }
+
+  table,
+  table tr td,
+  table tr th {
+    page-break-inside: avoid;
+  }
+
+  .utrecht-scroll-to-top-button {
+    display: none;
+  }
+
+  .utrecht-kcm-survey {
+    display: none !important;
+  }
+
+  [class*="map"],
+  [class*="kaart"],
+  iframe[src*="maps.google"],
+  iframe[src*="openstreetmap"] {
+    display: block !important;
+  }
+
+  nav,
+  [role="navigation"],
+  [class*="navigation"],
+  [class*="menu"],
+  [class*="breadcrumb"],
+  aside,
+  [class*="sidebar"],
+  .utrecht-skip-link,
+  .utrecht-kcm-survey,
+  .utrecht-scroll-to-top-button,
+  .utrecht-button-group,
```

Comment:
Please remove the `.utrecht-button-group` selector from the print stylesheet.
Hiding this element prevents links and call-to-action buttons inside it from appearing in the printed version.